prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
Hi Joseph, clicking on this link should show the index page, but i don't believe this does that. try changing the `href` value to `../index.html`